app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+
+  def last_three_posts
+    posts.where(user_id: 3).last(3)
+  end
```

Comment:
- [ ] Kindly make sure the `last_three_posts` method returns 3 most recent posts for any user and not only for user where `user_id: 3`. I suggest you use this approach: ``posts.order('created_at Desc')...``